Sources/TuistCore/Automation/XcodeBuildArgument.swift

```diff
@@ -18,6 +18,9 @@ public enum XcodeBuildArgument: Equatable, CustomStringConvertible {
     /// To override build settings.
     case buildSetting(String, String)
 
+    /// To pass additional arguments
+    case xcarg(String, String)
```

Comment:
I think we can remove `buildSetting` in favour of this one. What do you think? Looking at the implementation of `arguments`, both return the same value `KEY=VALUE`.

Reply:
Great!  Also thought about it